packages/lesswrong/server/callbacks/sunshineCallbackUtils.tsx

```diff
@@ -23,7 +28,71 @@ export async function triggerReviewIfNeeded(userId: string) {
   } else if (snoozed) {
     const contentCount = getCurrentContentCount(user);
     needsReview = contentCount >= user.snoozedUntilContentCount;
+  } else if (newModeAction && forumTypeSetting.get() === 'LessWrong') {
+    needsReview = true;
   }
 
   void Users.rawUpdateOne({ _id: user._id }, { $set: { needsReview: needsReview } });
 }
+
+function isNetDownvoted(comment: DbComment) {
+  return comment.baseScore <= 0 && comment.voteCount > 0;
+}
+
+function areLowQualityComments(comments: DbComment[]) {
+  const threshold = 2;
+  const downvotedCommentCount = comments.filter(isNetDownvoted).length;
+
+  return downvotedCommentCount >= threshold;
+}
+
+function areMediocreQualityComments(comments: DbComment[]) {
```

Comment:
Might prefer a name like "commentsHaveLowAverageKarma" or something that is transparent about happens

Reply:
(not certain about this one)